pkg/kn/commands/configuration_edit_flags.go

```diff
@@ -55,5 +75,47 @@ func (p *ConfigurationEditFlags) Apply(config *servingv1alpha1.ConfigurationSpec
 	if err != nil {
 		return err
 	}
+	limitsResources, err := p.computeLimitsResources()
+	if err != nil {
+		return err
+	}
+	requestsResources, err := p.computeRequestsResources()
+	if err != nil {
+		return err
+	}
+	err = servinglib.UpdateResources(config, requestsResources, limitsResources)
+	if err != nil {
+		return err
+	}
 	return nil
 }
+
+func (p *ConfigurationEditFlags) computeRequestsResources() (corev1.ResourceList, error) {
+	requestsCPU, err := resource.ParseQuantity(p.RequestsFlags.CPU)
+	if err != nil {
+		return corev1.ResourceList{}, err
+	}
+	requestsMemory, err := resource.ParseQuantity(p.RequestsFlags.Memory)
+	if err != nil {
+		return corev1.ResourceList{}, err
+	}
+	return corev1.ResourceList{
+		corev1.ResourceCPU:    requestsCPU,
+		corev1.ResourceMemory: requestsMemory,
+	}, nil
+}
+
+func (p *ConfigurationEditFlags) computeLimitsResources() (corev1.ResourceList, error) {
+	limitsCPU, err := resource.ParseQuantity(p.LimitsFlags.CPU)
+	if err != nil {
+		return corev1.ResourceList{}, err
+	}
+	limitsMemory, err := resource.ParseQuantity(p.LimitsFlags.Memory)
+	if err != nil {
+		return corev1.ResourceList{}, err
+	}
+	return corev1.ResourceList{
```

Comment:
I would like to see this take in whatever the previous resource list was -- if you change CPU resources with an update, you should expect to leave memory the same.

Reply:
Yeah, that's fair. My bad

Reply:
I am thinking of something like:

```go
       [...]
        var err error
	var requestsCPU, requestsMemory resource.Quantity
	if p.RequestsFlags.CPU != "" {
		requestsCPU, err = resource.ParseQuantity(p.RequestsFlags.CPU)
		if err != nil {
			return corev1.ResourceList{}, err
		}		
	} else {
		requestsCPU = config.RevisionTemplate.Spec.Container.Resources.Requests[corev1.ResourceCPU]
	}
       [...]
```